tests/objects/test_types.py

```diff
@@ -0,0 +1,40 @@
+import pytest
+
+import strawberry
+from strawberry.exceptions import ObjectIsNotClassError
+
+
+def test_raises_error_when_using_type_with_a_not_class_object():
+    expected_error = (
+        r"strawberry.type can only be used with class types. Provided "
+        "object .* is not a type."
```

Comment:
It doesn't end up changing things here, but you probably wanted to do this to each of these expected errors:
```suggestion
        r"strawberry.type can only be used with class types. Provided "
        r"object .* is not a type."
```

Reply:
Yeah the match is working so think python is converting them to be a rstring both... but agree, it's also looks better in the IDE :)

Reply:
It's working because the second string has no special characters. Note that `r""` doesn't make a regex string, it makes a _raw_ string, where escape characters (i.e. those prefixed by `\`s) are ignored and interpreted literally. However, people often use `r""` strings to indicate regex patterns even if not using special characters

Reply:
Oh, you are right, I can omit now the `r`... Was not working at some point, so I added the `r`... I gues there was something else wrong :D

Reply:
I tend to use the `r""` strings for regex even if there's no escape sequences. I think it adds a little expressiveness to the code saying "hey, this is a pattern used as a regex". Could help future maintainers. But I'll leave the choice to use it or not up to you (or @patrick91 if he has an opinion :P)

Reply:
Yeah totally agree! I will leave it there :)